content/docs/reference-javascript-environment-requirements.md

```diff
@@ -18,13 +18,13 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 
 ReactDOM.render(
-  <h1>Hello, world!</h1>,
+  <h1>Olá, mundo!</h1>,
```

Comment:
> ## Glossary and Style Guide
> ...
> * Leave code blocks untranslated. (It is usually OK to translate comments inside them.)

Reply:
thanks for the input. I think for this case specifically it's ok to translate strings inside HTML/JSX elements as it doesn't change any structure/logic. I'll update the glossary